test/unit/spec/narrative_core/upload/stagingAreaViewer-spec.js

```diff
@@ -260,7 +260,7 @@ define([
             const placeholder = $container.find('span.select2-selection__placeholder').html();
             expect(placeholder).toContain('Select a type');
 
-            //The options that should be in the import as dropdown
+            // The options that should be in the import as dropdown
```

Comment:
This has tweaks to keep up with the function changes. Also adds a test for the `Upload with URL` button.